apispec/openapi.json

```diff
@@ -5118,7 +5118,7 @@
           }
         },
         "operationId": "get-report-requests",
-        "description": "The API is to query the total request count for all of customer’s transports.  Parameters allow you to specify a time period, an interval type.",
+        "description": "The API is to query the total request count for all of the customer’s transports.  Parameters allow you to specify a time period and an interval type.",
```

Comment:
Replace the first sentence with "Return the total request count for all transports."